eclass/kernel-build.eclass

```diff
@@ -195,6 +195,11 @@ kernel-build_src_install() {
 	local image_path=$(dist-kernel_get_image_path)
 	cp -p "build/${image_path}" "${ED}/usr/src/linux-${ver}/${image_path}" || die
 
+	# Install the unstripped uncompressed vmlinux for use with systemtap
+	# etc.  Use mv rather than doins for the same reason as above --
+	# space and time.
+	use debug && mv build/vmlinux "/usr/src/linux-${ver}/"
```

Comment:
`mv` needs a `|| die`, and so please use full `if use debug; then ...` here.

Reply:
Done on merge.